code/modules/keybindings/bindings_atom.dm

```diff
@@ -22,3 +22,5 @@
 		keybind_face_direction(movement_dir)
 	else
 		user?.Move(get_step(src, movement_dir), movement_dir)
+
+	return !!movement_dir //true if there was actually any player input
```

Comment:
recorded input and doing the actually expensive part of the move are different things, no?

Reply:
This is just to record subsystem stats for number of actual client movement inputs per second